File: Assignments/2_Operators and Expression/C#/Q9.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Enter the number of seconds: ");
        int totalSeconds = int.Parse(Console.ReadLine());

        int hours = totalSeconds / 3600;
        totalSeconds %= 3600;
        int minutes = totalSeconds / 60;
        int seconds = totalSeconds % 60;

        Console.WriteLine($"{hours} hrs {minutes} mins {seconds} secs");
    }
}

File: Assignments/2_Operators and Expression/Javascript/Solutions/Q9.js
function convertSeconds(seconds) {
    const hours = Math.floor(seconds / 3600);
    seconds %= 3600;
    const minutes = Math.floor(seconds / 60);
    seconds %= 60;
    return { hours, minutes, seconds };
}

const totalSeconds = 7802;
const result = convertSeconds(totalSeconds);
console.log(`${result.hours} hrs ${result.minutes} mins ${result.seconds} secs`);
